example_app_generator/generate_app.rb

```diff
@@ -1,7 +1,7 @@
 rspec_rails_repo_path = File.expand_path("../../", __FILE__)
 rspec_dependencies_gemfile = File.join(rspec_rails_repo_path, 'Gemfile-rspec-dependencies')
 rails_dependencies_gemfile = File.join(rspec_rails_repo_path, 'Gemfile-rails-dependencies')
-bundle_install_path = File.join(rspec_rails_repo_path, '..', 'bundle')
+bundle_install_path = File.join(rspec_rails_repo_path, 'vendor', 'bundle')
```

Comment:
Yeah, I find this setup weird. My `code` dir, where this setup installs gems, contains code I'm working on, not gem installations.

Anyways, removing this.